composer.json

```diff
@@ -22,6 +22,6 @@
         }
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.5"
+        "phpunit/phpunit": "^5.0 || ^6.0 || ^7.0.15"
```

Comment:
This version range is required to get tests to pass on all versions of PHP in the travis matrix.